ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/accordion/v1/accordion/accordion.html

```diff
@@ -29,10 +29,12 @@
      data-cmp-single-expansion="true"
      data-placeholder-text="${wcmmode.edit && 'Please drag Accordion item components here' @ i18n}">
 
-    <div data-sly-call="${label.label @componentId=accordion.id, labelValue=accordion.label.value, labelVisible=accordion.label.visible, bemBlock='cmp-accordion'}"
-         data-sly-unwrap></div>
-    <div data-sly-call="${questionMark.questionMark @componentId=accordion.id, longDescription=accordion.description, bemBlock='cmp-accordion'}"
-         data-sly-unwrap></div>
+    <div class="cmp-accordion__label-container">
```

Comment:
Shouldn't it be cmp-adaptiveform-accordion?
@rismehta Seems 'cmp-accordion' is being used a lot of places which I think is incorrect. Should we fix it everywhere

Reply:
We are re-using the accordion component of the site team. Please check here
https://github.com/adobe/aem-core-forms-components/blob/master/ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/accordion/v1/accordion/accordion.html#L27